include/zephyr/bluetooth/bluetooth.h

```diff
@@ -2849,7 +2857,9 @@ int bt_le_filter_accept_list_clear(void);
 /**
  * @brief Set (LE) channel map.
  *
- * @param chan_map Channel map.
+ * Used to inform the Controller of which RF channels to use for LE communication with the Host.
+ *
+ * @param chan_map Channel map. 5 bytes where each bit represents a channel.
```

Comment:
Suggest to document that only the first 37 bits are valid 
![image](https://github.com/user-attachments/assets/d0c0e1d7-1c90-44cb-8d08-9b6093c65e46)